ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -31,16 +33,35 @@ const ScrollToBottom = ({ children }: ContentProps) => {
   const t = useContext(I18nContext);
   const dispatch = useDispatch();
   const currentConfirmation = useSelector(currentConfirmationSelector);
+  const previousId = usePrevious(currentConfirmation?.id);
 
   const {
     hasScrolledToBottom,
     isScrollable,
     isScrolledToBottom,
     onScroll,
     scrollToBottom,
+    setHasScrolledToBottom,
     ref,
   } = useScrollRequired([currentConfirmation?.id]);
 
+  /**
+   * Scroll to the top of the page when the confirmation changes. This happens
+   * when we navigate through different confirmations. Also, resets hasScrolledToBottom
+   */
+  useEffect(() => {
+    if (isEqual(previousId, currentConfirmation?.id)) {
+      return;
+    }
+
+    setHasScrolledToBottom(false);
+
+    const scrollTo = (ref?.current as null | HTMLDivElement)?.scrollTo;
```

Comment:
Would it be cleaner to encapsulate this and the above `setHasScrolledToBottom` inside the `useScrollRequired` hook in a method such as `scrollToTop` or `reset`?

Maybe we could rename `useScrollRequired` to `useScroll` to empower it?

Reply:
I like the idea of renaming useScrollRequired to useScroll. There are 3 other instances that use `useScrollRequired` (relating to snaps). Because of this, I don't think it would be a good idea to move this logic inside `useScrollRequired` at this time. I think it'd be better to consider this after we have more use-cases. 

 we could rename the file in a separate PR

Reply:
We could make a separate hook that uses the existing hook to avoid any risk, but not urgent given time constraints.